recommend/policy.go

```diff
@@ -37,6 +37,7 @@ func addPolicyRule(policy *types.KubeArmorPolicy, r Rules) {
 			dirRule := types.KnoxMatchDirectories{
 				Dir:        path,
 				Recursive:  true,
+				OwnerOnly:  true,
```

Comment:
By default, this should be false.
```suggestion
				OwnerOnly:  false,
```

Reply:
@nyrahul  When using `OwnerOnly: false`  the entire `ownerOnly` part is getting omitted in the policy YAML. From my research, I could see that to include false boolean data in the output file,  we have to use pointer booleans [https://stackoverflow.com/questions/37756236/json-golang-boolean-omitempty]

Reply:
@nyrahul The current code will add the ownerOnly flag if it's enabled in Rules.json. I hope this will address the issue we had.
1. Output with ownerOnly: true in Rules.json
    ```
      cat out/alpine-latest/sys-bin-protect.yaml
      apiVersion: security.kubearmor.com/v1
      kind: KubeArmorPolicy
      metadata:
        name: sys-bin-protect
      spec:
        action: Audit
        file:
          matchDirectories:
          - dir: /bin/
            ownerOnly: true
            recursive: true
          - dir: /usr/bin/
            ownerOnly: true
            recursive: true
          - dir: /usr/sbin/
            ownerOnly: true
            recursive: true
        message: attempted access to system binaries
        selector:
          matchLabels:
            mysql: testdata
            test: labels
            testvalue: isgreat
        severity: 1
        tags:
        - CIS
        - MITRE
     ```
2. Output without `ownerOnly: true` in Rules.json
    ```
    cat out/alpine-latest/maint-tool-access.yaml
    apiVersion: security.kubearmor.com/v1
    kind: KubeArmorPolicy
    metadata:
      name: maint-tool-access
    spec:
      action: Audit
      file:
        matchDirectories:
        - dir: /sbin/
          recursive: true
      message: restricted maintenance tool access attempted
      selector:
        matchLabels:
          mysql: testdata
          test: labels
          testvalue: isgreat
      severity: 1
      tags:
      - PCI-DSS
      - MITRE
    ```